shub/apps/base/views/main.py

```diff
@@ -44,7 +44,8 @@ class VersionView(APIView):
 
     renderer_classes = (JSONRenderer,)
 
-    def get(self, request):
+    @staticmethod
```

Comment:
Please don't add static methods to this (and all following) APIViews. https://stackoverflow.com/questions/49992869/use-staticmethod-in-django-rest-framework-endpoint-method